src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReportService {
```

Comment:
Nice naming. Pls, rename other INTERFACES due to naming conventions. They shouldn't be VERBS.